### DISCUSSION TITLE
Volunteer Contribution Inquiry

### DISCUSSION THREAD

**Discussion by mint673** [robolectric/robolectric#9288] [General]
Hi, I'm MinTing, a 4th year CS student. I'm looking to further my knowledge in Java and mobile by contributing to prod-level code. I came across your org from GSOC and found what you do really interesting.

I am aware that this year's GSOC is concluding, so I'm wondering if you're accepting volunteers outside of GSOC right now and how can one contribtue? Is a volunteer allowed to work on a project from your ideas list and participate in any scrum/meetings you have regularly?

**Comment by MGaetan89** [comment]
Hey @mint673,

Thanks for your interest in this project! Robolectric welcomes contribution from everyone 🙂

You can check the [Building And Contributing](https://github.com/robolectric/robolectric/tree/master?tab=readme-ov-file#building-and-contributing) section of the readme to know how to setup the project on your machine.
Then, you can check the issues labeled https://github.com/robolectric/robolectric/labels/starter%20issue and https://github.com/robolectric/robolectric/labels/help%20wanted for inspiration for a first contribution.

This is for the getting started part. For your questions specifically linked to GSoC projects, I'll let the appropriate people answer it.

Good luck 🙂
